extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fsinternetradio"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="radio">
+        <label>Internet Radio</label>
+        <description>An internet radio device based on the Frontier Silicon chipset.</description>
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="mode" typeId="mode"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="play-info-name" typeId="play-info-name"/>
+            <channel id="play-info-text" typeId="play-info-text"/>
+            <channel id="preset" typeId="preset"/>
+        </channels>
+
+        <config-description>
+            <parameter name="ip" type="text" required="true">
+                <label>Internet Radio Address</label>
```

Comment:
Remove the "Internet Radio" from all labels (config as well as channels), please - labels should be as short as possible.